.github/workflows/integration-tests.yaml

```diff
@@ -14,7 +14,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v2
         with:
-          go-version: ^1.20
+          go-version: 1.20.5
```

Comment:
I think this would be better if it was set to https://github.com/marketplace/actions/setup-go-environment#getting-go-version-from-the-gomod-file